package.json

```diff
@@ -4,10 +4,11 @@
   "description": "Helper module for atom linter providers",
   "main": "lib/index.js",
   "scripts": {
+    "lint": "(flow check) && (eslint .)",
     "test": "apm test",
-    "lint": "eslint .",
-    "watch": "ucompiler watch",
-    "compile": "ucompiler go"
+    "clean": "rm -rf lib",
```

Comment:
It would be nice if this was cross platform, but I can't find any way to do that without introducing a dependency 😕.